### PR TITLE
Enhancement (ci): Add validation for `PIPELINE`

### PR DESCRIPTION
**Pull request by leojonathanoh**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -234,6 +234,9 @@ elif [ "$PIPELINE" = 'update' ]; then
         "$BUILD_CONTEXT"
     docker tag "$GAME_IMAGE" "$GAME_IMAGE_LAYERED"
     date
+else
+    echo "Unknown PIPELINE=$PIPELINE"
```
Comment (joeltimothyoh) [inline build.sh:238]: how about this

```
    echo "Invalid PIPELINE '$PIPELINE'"
```

**Review by joeltimothyoh — CHANGES_REQUESTED** [review]

File: build.sh (hunk 1 above)
Comment (leojonathanoh) [reply build.sh:238]: whichever, doesn't really matter.

Comment (joeltimothyoh) [reply build.sh:238]: also, this condition should be moved to the validation section in https://github.com/startersclan/docker-sourceservers/pull/132/files#diff-4d2a8eefdf2a9783512a35da4dc7676a66404b6f3826a8af9aad038722da6823R157

Comment (leojonathanoh) [reply build.sh:238]: yes

**Comment by leojonathanoh** [comment]
lgtm

**Pull request merged**